### PR TITLE
docs(context-selector): Update v6 images.

### LINKED ISSUE #4306: Guideline updates: Context selector
This will also close https://github.com/patternfly/patternfly-org/issues/3679

### PR DESCRIPTION
**Pull request by edonehoo**
Closes #4306 and https://github.com/patternfly/patternfly-org/issues/3679

### CODE REVIEW

File: packages/documentation-site/patternfly-docs/content/design-guidelines/components/context-selector/img/context-selector-search.svg
Comment (andrew-ronaldson) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/components/context-selector/img/context-selector-search.svg:1]: The caret on this menu toggle looks broken. It should be aligned right of the toggle container. May be a bug with our Figma component

File: packages/documentation-site/patternfly-docs/content/design-guidelines/components/context-selector/img/page-context-selector.svg
Comment (andrew-ronaldson) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/components/context-selector/img/page-context-selector.svg:1]: Is there a file size difference between pngs and svgs? If not SVGs would be better as they won't rasterize and we don't have to worry about the pixels as much

**Review by andrew-ronaldson — COMMENTED** [review]
Just a couple comments

File: packages/documentation-site/patternfly-docs/content/design-guidelines/components/context-selector/img/page-context-selector.svg
Comment (edonehoo) [reply packages/documentation-site/patternfly-docs/content/design-guidelines/components/context-selector/img/page-context-selector.svg:1]: I think I managed to swap all the pngs out for svgs for that reason!

File: packages/documentation-site/patternfly-docs/content/design-guidelines/components/context-selector/img/context-selector-search.svg
Comment (edonehoo) [reply packages/documentation-site/patternfly-docs/content/design-guidelines/components/context-selector/img/context-selector-search.svg:1]: Ah yes, ty! That's correct - the Figma component isn't right aligning the toggle. I've been fixing it manually, but missed this one

**Review by andrew-ronaldson — APPROVED** [review]
magnifique!

**Pull request merged**